docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
       - "extensions:/opt/sonarqube/extensions"
       - "logs:/opt/sonarqube/logs"
       - "data:/opt/sonarqube/data"
-
+    restart: unless-stopped
```

Comment:
what is the use case for this need ?
for me, if container crashes, the developer must check why on logs et make a correction or an issue for correction.
if we add this option, we could miss the real pb